src/index.js

```diff
@@ -1 +1,91 @@
 import './style.css';
+
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+// todo: ===========================================> create game
+async function postGame() {
+  const temp = await fetch(baseUrl, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      name: '<----- Saman"s game ----->',
+    }),
+  });
+  return temp.json();
+}
+
+async function separateGameId(arg) {
+  const temp = arg.result.split(' ')[3];
+  return temp;
+}
+
+async function whenStart() {
+  const gameCreated = await postGame();
+  const seperatedIdPromise = separateGameId(gameCreated);
+  return seperatedIdPromise;
+}
+
+const gameIdPromise = whenStart();
+
+async function getScore() {
+  const currentId = await gameIdPromise;
+  const temp = await fetch(`${baseUrl}${currentId}/scores/`);
```

Comment:
- [ ] Kindly change these functions to es6 arrow functions, as required under the [JavaScript best practices guide](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.) to use es6 syntaxes.